shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -517,6 +523,48 @@ - (NSString*)accessibilityLabel {
   return label;
 }
 
+- (bool)semanticsObjectContainsPoint:(SemanticsObject*)semanticsObject withPoint:(CGPoint)point {
+  const SkRect& rect = [semanticsObject node].rect;
+  CGRect localRect = CGRectMake(rect.x(), rect.y(), rect.width(), rect.height());
+  CGRect globalRect = ConvertRectToGlobal(semanticsObject, localRect);
+  return CGRectContainsPoint(globalRect, point);
+}
+
+- (double)getSize:(SemanticsObject*)semanticsObject {
+  const SkRect& rect = [semanticsObject node].rect;
+  return rect.width() * rect.height();
+}
+
+- (SemanticsObject*)search:(SemanticsObject*)semanticsObject withPoint:(CGPoint)point {
+  if ([semanticsObject children].count == 0) {
+    if ([self semanticsObjectContainsPoint:semanticsObject withPoint:point] &&
+        [self isFocusable:semanticsObject]) {
+      return semanticsObject;
+    }
+    return nil;
+  }
+
+  SemanticsObject* bestChild = nil;
+  for (SemanticsObject* child in [semanticsObject children]) {
```

Comment:
This logic could use a comment.

Reply:
Comment also added to the `_accessibilityHitTest` function.